src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -350,7 +350,8 @@ export const FileUpload:React.FC<IFileUploadProps>=({
     useLayoutEffect(() => {
         function updateSize() {
             if(rootRef.current?.clientWidth){
-                dispatch({type:SET_COMPONENTWIDTH,value:rootRef.current.clientWidth-state.margin*2-state.padding*2})
+                const width=rootRef.current.clientWidth-state.margin*2-state.padding*2
```

Comment:
Needs better naming, what is this calculation performing? Descriptive is better than short.